src/classes/CW_AudienceCSVProcessHelper.cls

```diff
@@ -0,0 +1,406 @@
+        		//CW_Audience_Group__c.NAme
+        		GroupsIdmap.Put(ToUpper(grpRec.Name),grpRec.ID);
+        }
+        
+        //Find FreebetID Select c.Reward_Catalogue__c, c.FreeBet_Id__c From CW_Reward_Library__c c
+        //for(CW_Reward_Library__c fbRec: [SELECT ID,FreeBet_Id__c FROM CW_Reward_Library__c WHERE FreeBet_Id__c=:FreeBetIdmap.KeySet() AND Reward_Catalogue__c= :CsvStatus.Reward_Catalogue__c ]){
+        //		FreeBetIdmap.Put(ToUpper(fbRec.FreeBet_Id__c),fbRec.ID);
+        //}
+        
+        for(CW_CSV_Stage_AudienceMembers__c rec:MembrRewardsMap.Values()){
+        		rec.Audience_Group_ref__c =GroupsIdmap.get(ToUpper(rec.Group_Name__c));
+        		rec.Contact_ref__c        =PlayerIdmap.get(ToUpper(rec.PlayerId__c));
+        		rec.Reward_Library__c     =FreeBetIdmap.get(ToUpper(rec.FreeBetId__c));
+        }
+        Database.insert(MembrRewardsMap.Values());
+    }
+    
+    
+   public void processCreateSheildAccounts( List<CW_CSV_Stage_AudienceMembers__c> scope){
+   	CsvStageList=scope;
+   	map<string,string> PlayrsMap= new map<string,string>();
+   	for(CW_CSV_Stage_AudienceMembers__c rec:CsvStageList ){
+   		PlayrsMap.Put(rec.PlayerId__c.ToUpperCase(),rec.PlayerId__c);
+   	}
+   	map<string,Id> PlayrsIdMap=CreateSheildAccounts(new Set<string>(PlayrsMap.values()));
+   	
+   	for(CW_CSV_Stage_AudienceMembers__c rec:CsvStageList ){
+   		rec.Contact_ref__c=PlayrsIdMap.get(rec.PlayerId__c.ToUpperCase());
+   	}
+   	update CsvStageList;
+   }
+   
+   //Step 2  - create Accounts and groups.
+   
+   public void processCreateAudienceGroups(List<CW_CSV_Stage_AudienceMembers__c> scope){
+   	CsvStageList=scope;
+   	map<string,string> groupMap= new map<string,string>();
+   	for(CW_CSV_Stage_AudienceMembers__c rec:CsvStageList ){
+   		groupMap.Put(rec.Group_Name__c.ToUpperCase(),rec.Group_Name__c);
+   	}
+   	map<string,Id> groupIdMap=CreateAudienceGroups(new Set<string>(groupMap.values()));
+   	
+   	for(CW_CSV_Stage_AudienceMembers__c rec:CsvStageList ){
+   		rec.Audience_Group_ref__c=groupIdMap.get(rec.Group_Name__c.ToUpperCase());
+   	}
+   	update CsvStageList;
+   }
+    
+    
+   public void processImportDataToCamp(List<CW_CSV_Stage_AudienceMembers__c> scope){
+   	//We have set groups , Members adn freeBets. So we can impotr all with seted reference others records a ignored.
+   	CsvStageList=scope;
+   	//firts step crate non exited camping rewards and load existings
+   	loadCreateCampaignReward();
+   	
+   	//Next step is  CreateAudiece sschedule
+   	createLoadAudienceGroupSchdule();
+   	
+   	createAudinceGroupScheduleReward();
+   	
+   	//And olmost FInala Step Is  
+   	createAudinceMember();
+   	//And Final step
+ 		createAudinceMemberReward();
+   } 
+    
+    
+   //Private methods for step 2 
+   private map<string,Id> CreateSheildAccounts(Set<String> ContactNameList){
```

Comment:
Style issue: methods should use the "lowerCamelCase" notation. Suggestion: rename accordingly.